src/contrib/serializers/Akka.Serialization.Hyperion/reference.conf

```diff
@@ -1,80 +1,80 @@
 ﻿akka.actor {
-	serializers {
-		hyperion = "Akka.Serialization.HyperionSerializer, Akka.Serialization.Hyperion"
-	}
-	serialization-settings {
-		hyperion {
-			# If true, hyperion serializer will preserve references between objects. 
-			# This is necessary to support things such as cyclic dependencies between objects.
-			preserve-object-references = true
+  serializers {
```

Comment:
Nothing is changed on the default config file, all changes were white space changes from tab to spaces to make sure that text formatting is consistent